deployment-examples/azure/serverless/extension/README.md

```diff
@@ -0,0 +1,15 @@
+## Deploy as Azure function app
+
+1. In order to setup the extension module config, follow our [how to run guide](../../../../extension/docs/HowToRun.md).
+2. Extension module requires `ADYEN_INTEGRATION_CONFIG` as `environment variable` or `.extensionrc` file to configure with [external file configuration](../../../../extension/docs/HowToRun.md#external-file-configuration).
```

Comment:
What is the reason of the file `.extensionrc` for azure?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/9a641f16cdbcf60dbd91701cc93d098faa79e0a1

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/addec52897da5c24a72102e9b6438dfcd40ca2d4

Removed rc file usage in Azure and AWS docs.